src/main/resources/assets/computercraft/lua/bios.lua

```diff
@@ -826,6 +826,17 @@ if term.isColour() then
     settings.set( "bios.use_multishell", true )
     settings.set( "list.file_color", "white" )
     settings.set( "list.directory_color", "green" )
+    settings.set( "edit.backgroundColor", "black" )
```

Comment:
Should these be `underscore_case` rather than `camelCase`? The other settings are, so it would be nice to be consistent.